api/openapi-spec/swagger.json

```diff
@@ -20800,7 +20800,7 @@
   },
   "info": {
     "title": "Kubernetes",
-    "version": "v1.18.0"
+    "version": "v1.17.0"
```

Comment:
revert this line, verify scripts will be unhappy (your local tags are not up-to-date, fyi)

Reply:
Sorry for all this back and forth...